src/frontend/src/drakulaAi3nd.js

```diff
@@ -0,0 +1,121 @@
+            if (potentialAction < 0.7) {
+                bumpNumber = true;
+            } else {
+                // Set both bumpValue and bumpNumber to true
+                bumpValue = true;
+                bumpNumber = true;
+            }
+        } else if (chosenValue < 4 && chosenNumber >= halfDice) {
+            //chosen number max 5 min 2
+            if (potentialAction < 0.6) {
+                bumpValue = true;
+            } else {
+                // Set call
+                call = true;
+            }
+        } else if (chosenValue >= 4 && chosenNumber < halfDice) {
+            //chosen number max 4 min 1, Value max 6 min 4
+            if (potentialAction < 0.6) {
+                bumpNumber = true;
+            } else {
+                // Set both bumpValue and bumpNumber to true
+                bumpValue = true;
+                bumpNumber = true;
+            }
+        } else if (chosenValue >= 4 && chosenNumber >= halfDice) {
+            //chosen number max 5 min 2
+            if (potentialAction < 0.3) {
+                bumpValue = true;
+            } else {
+                // Set call
+                call = true;
+            }
+        } else {
+            call = true;
+        }
+    }
+    // Less dice drakula less bump
+    function drakulaEqualOrLessDice() {
+        if (potentialAction < 0.4) {
+            call = true; // Call more often in a defensive stance
+        } else if (chosenValue <= 3 && potentialAction < 0.5) {
+            bumpValue = true;
+        } else if (chosenNumber < totalDice && potentialAction < 0.9) {
+            bumpNumber = true;
+        } else {
+            call = true; // Fall back to calling if no other option is favorable
+        }
+    }
+
+    // When Dracula has more Dice
+    if (draculaDice.length > playerDice.length) {
+        draculaMoreDice();
+        // When Dracula has fewer or equal number of dices than player
+    } else if (draculaDice.length <= playerDice.length) {
+        drakulaEqualOrLessDice();
+    } else {
+        call = true; // Unexpected situation meet, Dracula calls
+    }
+
+
+    let draculaAction = null;
+    if (call) {
+        draculaAction = 'call';
+    } else if (bumpValue && bumpNumber) {
+        draculaAction = 'bumpValueAndNumber';
+    } else if (bumpValue) {
+        draculaAction = 'bumpValue';
+    } else if (bumpNumber) {
+        draculaAction = 'bumpNumber';
+    } else {
+        draculaAction = 'call';
+    }
+
+    // Handle the actions
+    if (draculaAction === 'call') {
+        //console.log(`Dracula chooses to call -> Number:${chosenNumber}, Value:${chosenValue}`);
+        return { action: 'call', chosenNumber, chosenValue };
+    }
+
+    if (draculaAction === 'bumpValueAndNumber') {
+        if (chosenValue < 6 && chosenNumber < 8) {
+            chosenValue += 1;
+            chosenNumber += 1;
+            //console.log(`Dracula bumps both number and value -> Number:${chosenNumber}, Value:${chosenValue}`);
```

Comment:
We don't need to keep the comments around. Once again, we can restore everything from Git. The main thing is that this comment complicates the code for other people, since when editing your code the other person would not know if they should uncomment it, delete it or move it.

Sure, it might be obvious in cases like this, where the comment is clear. But in general this is not very appreciated.

Reply:
let's forget drakulaAi3nd.js